lib/node_modules/@stdlib/stats/base/dists/normal/ctor/benchmark/benchmark.js

```diff
@@ -324,17 +384,23 @@ bench( pkg+':skewness', function benchmark( b ) {
 bench( pkg+':variance', function benchmark( b ) {
 	var sigma;
 	var dist;
+	var len;
 	var mu;
+	var x;
 	var y;
 	var i;
 
 	mu = 2.0;
 	sigma = 3.0;
 	dist = new Normal( mu, sigma );
-
+	len = 100;
+	x = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		x[ i ] = uniform( EPS, 100.0 );
+	}
 	b.tic();
```

Comment:
```suggestion
	}
	
	b.tic();
```

This applies throughout the PR